kratos/geometries/geometry.h

```diff
@@ -1221,7 +1221,8 @@ class Geometry
             }
         } else if (LumpingMethod == LumpingMethods::DIAGONAL_SCALING) {
             IntegrationMethod integration_method = GetDefaultIntegrationMethod();
-            integration_method = static_cast<IntegrationMethod>(static_cast<int>(integration_method) + 1);
+            int j = std::min(static_cast<int>(integration_method) + 1, 4);
```

Comment:
```suggestion
            const int j = std::min(static_cast<int>(integration_method) + 1, 4);
```
Only because of this (modify geometry.h) we will need the approval from @KratosMultiphysics/technical-committee

Reply:
I used this as ad-hoc fix. I am using Gauss_5 for Triangle2D15. In that part (in geometry.h), in order to get higher accuracy, it adds one order more to Gauss, and tries to make it Gauss_6 (in order to calculta N^2). However, this option is not available in the code.
Another remedy for this is to extent Gauss points to higher orders. Do you think it will be a more suitable option?